openvpn/common/hexstr.hpp

```diff
@@ -180,7 +180,7 @@ namespace openvpn {
 	dest.push_back((high<<4) + low);
       }
     if (i != len)
-      throw parse_hex_error(); // straggler char      
+      throw parse_hex_error(); // straggler char
```

Comment:
code cleanups and style fixes should always go in their own commits and not merged with other changes.